examples/ex9.cpp

```diff
@@ -127,6 +134,12 @@ int main(int argc, char *argv[])
                   "Use binary (Sidre) or ascii format for VisIt data files.");
    args.AddOption(&vis_steps, "-vs", "--visualization-steps",
                   "Visualize every n-th timestep.");
+   args.AddOption((int*) &fem_type, "-fem", "--fem_type",
+                  "Choose fem discretization (CG or DG).");
```

Comment:
It is a bit confusing for the user here that DG = 0, CG = 1. Making this more explicit could help.